classes/class-admin.php

```diff
@@ -687,6 +690,94 @@ public function render_list_table() {
 	<?php
 	}
 
+	public function maybe_render_csv_page() {
+		$page = wp_stream_filter_input( INPUT_GET, 'page' );
+		$output_csv = wp_stream_filter_input( INPUT_GET, 'output_csv' );
+		if (  '1' !== $output_csv || 'wp_stream' !== $page ) {
```

Comment:
Rather than using the parameter `output_csv` with a value of `1`, can we make this more extendable by using: `output` with a value of `csv`? Might mean some refactoring.